### PR TITLE
Gc

### PR DESCRIPTION
**Pull request by AkhilYadavPadala**
@OfficialAhmed  @NitkarshChourasia 
I made changes.
Review my code :)

### CODE REVIEW

**Comment by NitkarshChourasia** [comment]
I will do by the morning afternoon of 7th October, 2023.

**Review by OfficialAhmed — CHANGES_REQUESTED** [review]
* Improper indentation from line **_86_** to **_101_** .Probably it won't interpret the code properly **_remove indentation_**.
* In line **__99__** `break` keyword is also indented incorrectly

_Double check the indentation please, the rest of the code works fine_

File: Calculator with simple ui.py [hunk 1]
```diff
@@ -33,16 +86,16 @@ def divide(x, y):
         num2 = float(input("Enter second number: "))
 
         if choice == '1':
-            print(num1, "+", num2, "=", add(num1, num2))
+            print(calculator.add(num1, num2))
 
         elif choice == '2':
-            print(num1, "-", num2, "=", subtract(num1, num2))
+            print(calculator.subtract(num1, num2))
```
Comment (NitkarshChourasia) [inline Calculator with simple ui.py:92]: Use a IDE if possible while coding.
Also, Use a code formatter if possible.
If possible, enable code formatter when saving the file.
You can also format the code by ```black```.
to install just type in cmd ```pip install black```
then format the file using cmd by typing in cmd ```black file_name.py```
and worry not, there are many ways to format a file. This clicks with me, that's it.

File: Calculator with simple ui.py [hunk 2]
```diff
@@ -1,23 +1,76 @@
 # Program make a simple calculator
 
-# This function adds two numbers
-def add(x, y):
-    return x + y
+class Calculator:
```
Comment (NitkarshChourasia) [inline Calculator with simple ui.py:3]: Can add a docsting for class, here...too.

File: Calculator with simple ui.py [hunk 3]
```diff
@@ -1,23 +1,76 @@
...
+calculator = Calculator()
 
-# This function divides two numbers
-def divide(x, y):
-    return x / y
 
 
-print("Select operation.")
 print("1.Add")
 print("2.Subtract")
```
Comment (NitkarshChourasia) [inline Calculator with simple ui.py:75]:
```python
calculator = Calculator()

# A dictionary that maps the choices to the functions
operations = {
    "1": calculator.add,
    "2": calculator.subtract,
    "3": calculator.multiply,
    "4": calculator.divide
}

# A loop that asks the user for input until they enter a valid choice
while True:
    print("Select operation.")
    print("1.Add")
    print("2.Subtract")
    print("3.Multiply")
    print("4.Divide")

    choice = input("Enter choice(1/2/3/4): ")

    # Check if the choice is valid
    if choice in operations:
        
        # Ask the user for two numbers
        num1 = float(input("Enter first number: "))
        num2 = float(input("Enter second number: "))

        # Get the function by its key from the dictionary
        operation = operations.get(choice)

        # Call the function with the two numbers and print the result
        result = operation(num1, num2)
        print(f"The result is {result}")

        # Break out of the loop
        break
    else:
        # Print an error message and continue the loop
        print("Invalid input. Please try again.")
```

Understand these snippet.

**Review by NitkarshChourasia — CHANGES_REQUESTED** [review]
Everything is perfect, just minor corrections and you are good to go.

**Comment by AkhilYadavPadala** [comment]
I made corrections on indentation @OfficialAhmed .
Thanks for  the suggestion @NitkarshChourasia . I used black command to do so

**Review by OfficialAhmed — APPROVED** [review]

**Comment by NitkarshChourasia** [comment]
> I made corrections on indentation @OfficialAhmed .
Thanks for the suggestion @NitkarshChourasia . I used black command to do so.

### Welcome.

**Comment by sarayusreeyadavpadala** [comment]
@geekcomputers Would you mind adding hacktoberfest and hacktoberfest-accepted labels to my pull request.

**Comment by AkhilYadavPadala** [comment]
@geekcomputers Would you mind adding hacktoberfest and hacktoberfest-accepted labels to my pull request.

**Comment by NitkarshChourasia** [comment]
This repo has not been added to that program-fest. @AkhilYadavPadala

**Pull request merged**